include/macros.h

```diff
@@ -16,6 +19,8 @@
 #define ARRAY_COUNT(arr) (s32)(sizeof(arr) / sizeof(arr[0]))
 #define ARRAY_COUNTU(arr) (u32)(sizeof(arr) / sizeof(arr[0]))
 
+#define SEGMENTED_TO_VIRTUAL(addr) (void*)((gSegments[SEGMENT_NUMBER(addr)] + K0BASE) + SEGMENT_OFFSET(addr))
```

Comment:
```suggestion
#define SEGMENTED_TO_K0(addr) (void*)((gSegments[SEGMENT_NUMBER(addr)] + K0BASE) + SEGMENT_OFFSET(addr))
```
Still prefer `SEGMENTED_TO_K0`

Reply:
Oh right, in MM we are moving to `SEGMENTED_TO_K0`. shorry